templates/profile-base.html

```diff
@@ -4,6 +4,20 @@
     % set nofollow = ['nofollow'] if participant.profile_nofollow else []
     % set noindex = ['noindex'] if participant.profile_noindex or 'amount' in request.qs else []
     % if nofollow or noindex
-    <meta name="robots" content="{{ ','.join(noindex + nofollow) }}" />
+    <meta name="robots" content="{{ ','.join(noindex + nofollow) }}">
     % endif
+    <meta name="description" content="{{ _("%s's profile in Liberapay" % participant.username ) }}">
+    <meta name="og:title" content="{{ participant.username }}">
+    <meta name="og:type" content="profile">
+    <meta name="og:url" content="{{ website.canonical_scheme + '://' + request.hostname + '/' + participant.username }}">
+    <meta name="og:image" content="{{ participant.avatar_url }}">
+    <meta name="og:description" content="???">
+    <meta name="og:locale" content="{{ locale.language }}">
+    % for l_completion, l_code, l_name, l_url in website.lang_list
+        <meta name="og:locale:alternate" content="{{ l_code }}">
+    % endfor
+    <meta name="og:site_name" content="Liberapay">
+    <meta name="profile:first_name" content="">
+    <meta name="profile:last_name" content="???">
```

Comment:
We don't store the birth names of users in our database, and we shouldn't display them publicly without consent.